config/drupal/phpcsfixer.rules.yml

```diff
@@ -68,7 +68,7 @@ parameters:
         indentation_type: false
         line_ending: true
         lowercase_keywords: true
-        class_attributes_separation: false
+        class_attributes_separation: true
```

Comment:
seems it should be an array nop ? 
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.35|fixer:class_attributes_separation

as we add it to replace `no_extra_blank_lines:tokens:use_trait, should we only specifiy trait ?

```
class_attributes_separation:
    - trait_import: none
```

Reply:
As trait_import is already none by default. I thought only putting `class_attributes_separation: true` allows to enable the rule without having to duplicate its default configuration.

Otherwise, yes, we juste need to copy the default configuration.

Reply:
okay, I will add a tests that ensure trait are handlede properly on `master` once done I wil (again) merge master here to see if it continue to works or if it fails :)